scripts/certs.sh

```
@@ -0,0 +1,23 @@
+echo "Generates a markdown file with the CNAME values for the certificates that are pending validation"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)